pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -303,6 +316,23 @@ def newpkgcb(pkgId, name, arch):
             result = await downloader.run()
             metadata_pb.increment()
 
+            repomd_path = result.path
+            repomd = cr.Repomd(repomd_path)
+
+            if optimize_sync:
+                if self.repository.latest_version().number > 1:
```

Comment:
All repository version numbers start with 0. So I think it should either be >= 1 or > 0.